contrib/node/install-sdn.sh

```diff
@@ -1,36 +1,31 @@
 #!/bin/bash
+set -x
 
 os::provision::install-sdn() {
-  local default_target="/usr"
-
   local deployed_root=$1
-  local target=${2:-${default_target}}
+  local binaries_path=$2
+  local target=${3:-}
+  local target_usrdir="${target}/usr"
+  local target_bindir="${target_usrdir}/bin"
+  local target_confdir="${target}/etc/cni/net.d"
+  local target_cnidir="${target}/opt/cni/bin"
 
-  if [ ! -d ${target} ]; then
-    mkdir -p ${target}
-  fi
+  mkdir -p -m 0755 "${target_usrdir}"
```

Comment:
I think I commented on these before but I still think `o+rwx,g+rx,o+rx` is infinitely more readable (also, why do we need to be this explicit? We don't want `g+w`?).